### PR TITLE
Update ipython-vs-shell-vs-scripts.md

### PR DESCRIPTION
**Pull request by nem035**
The insight was a bit wordy so I trimmed it down.

I moved the text showcasing iPython vs Python terminals in the footnote and added a "why" explanation for the multi-line feature.

I also think we put too much emphasis on shells/scripts/terminal and done so too late. So I added some of those definitions as footnotes to the previous insight (see See also: https://github.com/enkidevs/curriculum/pull/2640) and simplified the language in this one.

### CODE REVIEW

**Comment by enki-hq** [comment]
### Changelog

| File Name | Error Message |
|:-:|:-:|
| Pull Request | Couldn't find a CHANGELOG entry for this pull request. |

> If you notice anything wrong with these rules contact andrei@enki.com.

**Comment by enki-hq** [comment]
### Changelog

| File Name | Error Message |
|:-:|:-:|
| Pull Request | Couldn't find a CHANGELOG entry for this pull request. |

> If you notice anything wrong with these rules contact andrei@enki.com.

**Comment by enki-hq** [comment]
### Changelog

| File Name | Error Message |
|:-:|:-:|
| Pull Request | Couldn't find a CHANGELOG entry for this pull request. |

> If you notice anything wrong with these rules contact andrei@enki.com.

File: python-data-analysis/python-data-analysis-core/analysis-environments/ipython-vs-shell-vs-scripts.md [hunk 1]
```diff
@@ -16,55 +16,33 @@ revisionQuestion:
...
-
-Terminals are interface sessions on the computer that we use to communicate with shells. We send input (commands) into and receive output (results) out of the shell via a terminal.
-
-The IPython interpreter, as well as the basic Python interpreter, are actually interactive shells that are accessed through the terminal via the `python`/`ipython` commands.
-
-Scripts are files with runnable code that you can use for easier execution, automation, and more. 
-
-So basically, we type code in a terminal that then sends that code to the command-line tool(shell) which then translates and executes that code for us.
+```sh
+python my_script.py
```
Comment (nem035) [inline python-data-analysis/python-data-analysis-core/analysis-environments/ipython-vs-shell-vs-scripts.md:44]:
```suggestion
# scripts with python code
# are usually saved with
# the .py extension
python my_script.py
```

**Comment by enki-hq** [comment]
### Changelog

| File Name | Error Message |
|:-:|:-:|
| Pull Request | Couldn't find a CHANGELOG entry for this pull request. |

> If you notice anything wrong with these rules contact andrei@enki.com.

**Review by Stefan-Stojanovic — APPROVED** [review]

**Comment by enki-hq** [comment]
### Insight previews:
- [ ] [enkidevs/curriculum/python-data-analysis/python-data-analysis-core/analysis-environments/ipython-vs-shell-vs-scripts.md](https://app.enkipro.com/public/preview-insight/enkidevs/curriculum/python-data-analysis/python-data-analysis-core/analysis-environments/ipython-vs-shell-vs-scripts.md?ref=1011403aabcce2125b3440f60ccd35304ec64372)

**Review by Stefan-Stojanovic — APPROVED** [review]
